build/azure-pipelines/darwin/sql-product-build-darwin.yml

```diff
@@ -136,16 +136,16 @@ steps:
       APP_NAME="`ls $APP_ROOT | head -n 1`"
       yarn smoketest --build "$APP_ROOT/$APP_NAME" --screenshots "$(build.artifactstagingdirectory)/smokeshots" --log "$(build.artifactstagingdirectory)/logs/darwin/smoke.log" --extensionsDir "$(build.sourcesdirectory)/extensions"
     displayName: Run smoke tests (Electron) (Continue on Error)
-    continueOnError: true
-    condition: and(succeeded(), and(eq(variables['RUN_TESTS'], 'true'), ne(variables['SMOKE_FAIL_ON_ERROR'], 'true')))
+    continueOnError: 'true'
```

Comment:
why is the value quoted?

Reply:
Oh whoops, that snuck in. I have schema validation enabled and it was complaining about it being a boolean - need to look into this since that seems like a bug.

Reply:
Filed an issue for that one - boolean seems to be the expected value (although I'm pretty sure it'll accept a string just fine. So maybe an issue in both places) https://github.com/redhat-developer/vscode-yaml/issues/799